pkgs/applications/misc/bemoji/default.nix

```
@@ -0,0 +1,31 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+}:
+
+stdenv.mkDerivation rec {
+  pname = "bemoji";
+  version = "0.3.0";
+
+  src = fetchFromGitHub {
+    owner = "marty-oehme";
+    repo = pname;
+    rev = version;
+    hash = "sha256-XXNrUaS06UHF3cVfIfWjGF1sdPE709W2tFhfwTitzNs=";
+  };
+
+  strictDeps = true;
+  dontBuild = true;
+
+  installPhase = ''
+    chmod +x bemoji
+    install -D bemoji $out/bin/bemoji
```

Install sets the executable bit by default, no need for `chmod`. Even if it didn't, if you're using `install` just use the `-m` flag to set the required bits.

---

```suggestion
    install -Dm555 bemoji -t $out/bin
```
and the `chmod` can be removed